core/assets/bundles/bundle_vi.properties

```diff
@@ -1726,9 +1726,9 @@ lenum.flying = Không quân.
 lenum.ground = Bộ binh.
 lenum.player = Quân lính do người chơi điều khiển.
 
-lenum.ore = Ore deposit.
-lenum.damaged = Damaged ally building.
-lenum.spawn = Enemy spawn point.\nMay be a core or a position.
+lenum.ore = Bãi Quặng.
+lenum.damaged = Công trình đồng minh hư hỏng.
+lenum.spawn = Khu vực hạ cánh hoặc lõi của kẻ địch.
```

Comment:
In our Vietnamese translation, we translate "Core" to "Căn cứ" instead of "Lõi".

Reply:
~~first time issue~~